docs/ci-cd-environment/self-hosted-agents-overview.md

```diff
@@ -24,13 +24,15 @@ Three different types of tokens are used by the agent to communicate with Semaph
 
 The Semaphore Toolbox offers a set of tools to navigate language versions, databases, cache, artifacts and checking out your code. Some of these features are tied to our hosted system, and will not be available on self-hosted agents.
```

Comment:
Not visible in this diff but I just noticed that this page has a H1 / main title only "Overview" -- pls change to be more specific like "Overview of self-hosted agents".